AGEaligner.cpp

```diff
@@ -66,14 +66,15 @@ bool AGEaligner::align(Scorer &scr,int flag)
 {
   _n_bpoints = 0; // Erase previous break points
   if (_len1 <= 0 || _len2 <= 0) return false;
```

Comment:
Distinguishing between "modes" and "display options" allows this change to forward the correct flags to the auxiliary aligner.